crates/gen/src/types/method.rs

```diff
@@ -230,29 +230,21 @@ impl Method {
             quote! {
                 pub fn #method_name<#constraints>(&self, #params) -> ::winrt::Result<#return_type> {
                     use ::winrt::AbiTransferable;
-                    match self.ptr.get_abi() {
-                        None => panic!("The `this` pointer was null when calling method"),
-                        Some(this) => {
-                            unsafe {
-                                let mut __ok: #return_type = ::std::mem::zeroed();
-                                (this.vtable().#method_name)(this, #args #return_arg)
-                                    .and_then(|| __ok )
-                            }
-                        }
+                    let this = self.ptr.get_abi().expect("The `this` pointer was null when calling method");
```

Comment:
Can't this be condensed down to "self.get_abi()" now?

Reply:
Yes it can. I can push that real quick.